source/stylesheets/site.css.scss

```diff
@@ -1579,33 +1579,18 @@ ul.showcase {
 
   width: 33.333%;
   margin: 10px 0;
-  &:nth-child(3n+1) {
-    a {
-      margin: 0 auto 0 0;
-    }
-  }
```

Comment:
If you want to go with this, you would also need to target `> *` instead of `a` for other screen sizes